services/submission.js

```diff
@@ -0,0 +1,129 @@
+const { NEW, IN_PROGRESS, SUBMITTED, RELEASED, COMPLETED, ARCHIVED} = require("../constants/submission-constants");
+const {v4} = require('uuid')
+const {getCurrentTime, subtractDaysFromNow} = require("../crdc-datahub-database-drivers/utility/time-utility");
+const {HistoryEventBuilder} = require("../domain/history-event");
+const {verifySession} = require("../verifier/user-info-verifier");
+const {getSortDirection} = require("../crdc-datahub-database-drivers/utility/mongodb-utility");
+const USER_CONSTANTS = require("../crdc-datahub-database-drivers/constants/user-constants");
+const ROLES = USER_CONSTANTS.USER.ROLES;
+const ALL_FILTER = "All";
+
+
+
+function listConditions(userID, userRole, aUserOrganization, params){
+    const validApplicationStatus = {status: {$in: [NEW, IN_PROGRESS, SUBMITTED, RELEASED, COMPLETED, ARCHIVED]}};
+    // Default conditons are:
+    // Make sure application has valid status
+    let conditions = [{$and: [validApplicationStatus]}];
```

Comment:
Could you think about using an object instead? It looks fine, but it might not be very intuitive at first. I think you no longer need an $or query after making adjustments
let condition = { ...validApplicationStatus };
const listAllApplicationRoles = ["ADMIN", "FEDERAL_LEAD", "CURATOR", "DC_POC"];
if (listAllApplicationRoles.includes(userRole)) {
    return [{ "$match": { condition } }];
}
.
.
.
condition["organization.name"] = aUserOrganization.orgName;
return [{ "$match": { condition } }];